sdk/search/azure-search-documents/src/main/java/com/azure/search/annotation/FieldIgnore.java

```diff
@@ -0,0 +1,20 @@
+package com.azure.search.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Marker annotation that indicates the field is to be ignored by converting to SearchField.
+ */
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldIgnore {
```

Comment:
Talking with @brjohnstmsft yesterday, it should be opt-out since that's how track 1 works. That said, by *not* attributing/annotating properties, you don't get much by default.